src/js/letters/utils/constants.js

```diff
@@ -20,10 +20,15 @@ export const GET_LETTER_PDF_SUCCESS = 'GET_LETTER_PDF_SUCCESS';
 export const UPDATE_BENFIT_SUMMARY_REQUEST_OPTION = 'UPDATE_BENFIT_SUMMARY_REQUEST_OPTION';
 
 export const UPDATE_ADDRESS = 'UPDATE_ADDRESS';
+
 export const SAVE_ADDRESS_PENDING = 'SAVE_ADDRESS_PENDING';
 export const SAVE_ADDRESS_SUCCESS = 'SAVE_ADDRESS_SUCCESS';
 export const SAVE_ADDRESS_FAILURE = 'SAVE_ADDRESS_FAILURE';
 
+export const GET_ADDRESS_COUNTRIES_SUCCESS = 'GET_ADDRESS_COUNTRIES_SUCCESS';
+export const GET_ADDRESS_COUNTRIES_FAILURE = 'GET_ADDRESS_COUNTRIES_FAILURE';
+export const GET_ADDRESS_STATES_SUCCESS = 'GET_ADDRESS_STATES_SUCCESS';
+export const GET_ADDRESS_STATES_FAILURE = 'GET_ADDRESS_STATES_FAILURE';
```

Comment:
I'm not sure what all the ui considerations are here, but would it make sense to have pending states as well?

Reply:
Good question. This fetch happens when the app initially loads, but the states and countries aren't loaded until the user clicks the "Update" button, so there's no lag from the user's perspective. I think it's fine without a pending state.